packages/cli/src/active-workflow-manager.ts

```diff
@@ -673,10 +673,21 @@ export class ActiveWorkflowManager {
 		const triggerFilter = (nodeType: INodeType) =>
 			!!nodeType.trigger && !nodeType.description.name.includes('manualTrigger');
 
+		// Retrieve unqique webhooks as some nodes have multiple webhooks
+		const workflowWebhooks = WebhookHelpers.getWorkflowWebhooks(
+			workflow,
+			additionalData,
+			undefined,
+			true,
+		);
+
+		const webhookNodeNames = workflowWebhooks.map((webhook) => webhook.node);
+		const uniqueWebhookCount = new Set(webhookNodeNames).size;
```

Comment:
```suggestion
		const uniqueWebhookCount = workflowWebhooks.reduce<Set<string>>((acc, webhook) => {
			acc.add(webhook.node)
			return acc
		}, new Set())
```

this is a silly change, the outcome is the same but in a single iteration.

up to you